includes/sanitizers/class-amp-meta-sanitizer.php

```diff
@@ -0,0 +1,246 @@
+<?php
+/**
+ * Class AMP_Links_Sanitizer.
+ *
+ * @package AMP
+ */
+
+/**
+ * Class AMP_Meta_Sanitizer.
+ *
+ * Sanitizes meta tags found in the header.
+ *
+ * @since 1.5.0
+ */
+class AMP_Meta_Sanitizer extends AMP_Base_Sanitizer {
+
+	/**
+	 * Tag.
+	 *
+	 * @var string HTML <meta> tag to identify and replace with AMP version.
+	 */
+	public static $tag = 'meta';
+
+	/**
+	 * Placeholder for default arguments, to be set in child classes.
+	 *
+	 * @var array
+	 */
+	protected $DEFAULT_ARGS = [ // phpcs:ignore WordPress.NamingConventions.ValidVariableName.PropertyNotSnakeCase
+		'use_document_element' => true, // We want to work on the header, so we need the entire document.
+	];
```

Comment:
Not sure if this will have any practical effect. Since we're not referencing `$this-args['use_document_element']` in the sanitizer, this is just setting a member which is going to be either unused or just overridden in the first place.

Reply:
Yes, I think this can be removed now.